data-manager/data-manager.js

```diff
@@ -1,12 +1,26 @@
-const doctors = require("../data/doctors-records");
-const patients = require("../data/patient-records");
+const doctorsRecords = require("../data/doctors-records");
+const patientsRecords = require("../data/patient-records");
 class DataManager {
+  getDateBase(string) {
+    switch (string) {
```

Comment:
Same as above. "string" as a parameter is ambiguous. You could use a variable name that describes what the parameter is.